main.js

```diff
@@ -0,0 +1,120 @@
+const API_KEY = "6d29bbab";
+
+document.getElementById("form-submit").addEventListener("submit", getIp);
+let page = 1;
+let searchWord = document.getElementById("searchWord");
+function getIp(e) {
+  e.preventDefault();
+  let iVal = searchWord.value;
+  search(iVal, page);
+}
+
+const renderMovieNotFound = (error = "Movie not found!") => {
+  let notFoundDiv = "";
+  notFoundDiv += `<div id="results"><p class="not__found">${error}</p></div>`;
+  document.getElementById("results").innerHTML = notFoundDiv;
+};
+
+async function search(ip, page) {
+  const url = `https://www.omdbapi.com/?s=${ip}&page=${page}&apikey=${API_KEY}`;
+
+  const response = await axios.get(url);
+  const data = response.data;
+  if (data.Response === "False") {
+    renderMovieNotFound(data.Error);
+  } else {
+    let output = "";
+    data.Search.forEach((result) => {
+      let poster = result.Poster ? result.Poster : "";
+      output += `<div class="col-md-3">
+          <div class="card">
+            <img class="card-img-top" src="${poster}" alt="Poster">
+            <div class="card-body">
+              <h6 class="card-title">${result.Title} ${result.Year}</h6>
+              <a class="btn btn-primary" id="${result.imdbID}">Show more</a>
+            </div>
+          </div>
+           </div>`;
```

Comment:
This peace still unrefactored

https://github.com/k1ddd/hillel-front-end-pro-/pull/14#discussion_r1028975230